core/store/src/db.rs

```diff
@@ -799,14 +799,14 @@ mod tests {
     #[test]
     fn test_prewrite_check() {
         let tmp_dir = tempfile::Builder::new().prefix("_test_prewrite_check").tempdir().unwrap();
-        let store = RocksDB::new(tmp_dir.path().to_str().unwrap()).unwrap();
+        let store = RocksDB::new(tmp_dir).unwrap();
         store.pre_write_check().unwrap()
     }
 
     #[test]
     fn test_clear_column() {
         let tmp_dir = tempfile::Builder::new().prefix("_test_clear_column").tempdir().unwrap();
-        let store = create_store(tmp_dir.path().to_str().unwrap());
+        let store = create_store(&tmp_dir.into_path());
```

Comment:
This feels weird -- the `.into_path()` clones the value, but `&` takes the reference. I'd expect `tmp_dir.path()` to work here?

Reply:
fixed